planning/behavior_path_planner/autoware_behavior_path_start_planner_module/src/clothoid_pull_out.cpp

```diff
@@ -1422,11 +1422,7 @@ std::optional<PullOutPath> ClothoidPullOut::plan(
     // STEP 5-7: Collision check
     // ===================================================================
     // Create PullOutPath for collision check
-    PullOutPath temp_pull_out_path;
-    temp_pull_out_path.partial_paths.push_back(clothoid_path);
-    temp_pull_out_path.start_pose =
-      clothoid_path.points.empty() ? start_pose : clothoid_path.points.front().point.pose;
-    temp_pull_out_path.end_pose = target_pose;
+    PullOutPath temp_pull_out_path{{clothoid_path}, {}, start_pose, target_pose};
```

Comment:
```suggestion
    const PullOutPath temp_pull_out_path{{clothoid_path}, {}, start_pose, target_pose};
```
?

Reply:
addressed in https://github.com/autowarefoundation/autoware_universe/pull/11311/commits/acb436b4a38600e9ac33b3a3fff531b294c934c7 :pray: